auditorium/src/views/components/auditorium/embed-code.js

```diff
@@ -59,17 +70,22 @@ const EmbedCode = (props) => {
       <Paragraph class='ma0 mb3'>
         {__('In case you are serving multiple domains from your Offen instance, please double check that the domain in this snippet matches the target account.')}
       </Paragraph>
+      <div class="flex items-end">
+        <div class="w-100 tr bb b--light-gray">
+          <button onClick={toggleScriptDisplay} class='pointer w-100 w-auto-ns fw1 f7 tc bn dib br1 ph2 pv1 black bg-black-10'><span class={classnames('ml2', 'dib', 'label-toggle', 'label-toggle', !useSnippetWithSRI? null : 'label-toggle--rotate')}></span> {buttonText}</button>
```

Comment:
With `classnames` you don't need to add ternary statements yourself, instead you can have the library conditionally apply classnames:

```js
classnames('a', 'b' {'label-toggle--rotate': useSnippetWithSRI })
```